locales/en.yml

```diff
@@ -16,6 +16,48 @@ en:
     tools-content: "Sketch, Craft, Adobe CC, Balsamiq, InVision, Flinto, Zeplin, Jira, Slack, Trello, Bootstrap, Ruby on Rails, Sass, WordPress, Bourbon."
     usability-testing: "Usability Testing"
     ux-design: "UX Design"
+  contact:
+    contact-content: "I am looking to contribute to a group of bold & talented individuals that are committed to user-centric design thinking and collaborative, agile approaches. If you think I may be a good fit for your team, please feel free to reach out. The best way to get a hold of me is by email "
+    thanks: "Thanks for visiting"
+  mh:
+    design-process: "Design Process"
+    design-process-content: "The culture I’ve worked to establish is that of highly rapid, transparent, and collaborative design and prototyping. We began with brainstorming workshops, low-fidelity sketches, wireframes, and then increased in detail as our vision became more defined. Additionally, we maintained a regular feedback loop with our users throughout the design process."
+    design-process-footnote: "Based on our user feedback, it was clear that the navigation of Mobile Heartbeat needed to improve. Through card sorting studies, we were able to simplify the navigation and organize related functionality into intuitive groupings on the bottom tab bar."
+    design-process-alt: "Sketch of Mobile Heartbeat information architecture."
+    discovery: "Discovery Phase"
+    discovery-content: "Our users have highly specific and complex workflows, and we needed to make sure to thoroughly research and understand them. Our research included shadowing, interviewing, focus groups & surveys. This ranged from observing a chaotic night shift in a Manhattan ER, to scrubbing into the OR to see how surgical teams communicate. As a result, we built a stronger understanding of the goals & frustrations of our customers."
+    discovery-footnote: "We chose to follow the Lean UX Canvas for initial problem definition to help us maintain focus and establish measurable outcomes. Through our process, we formulated three hypotheses to measure against. In addition to a complete UI redesign of the application, we focused on solving the core problems around our information architecture, physician workflow, and speed of urgent communications."
+    discovery-alt: "Discovery process using UX Lean Canvas"
+    heading: "Mobile Heartbeat"
```

Comment:
I'd like to see more like this when it comes to standardization of how you're declaring sub-categories in your YAML. It helps when authoring content, so you don't have to go digging around for the key for a string, which kinda defeats the purpose of using structured data like this.

Keeping keys generic to use terms like "title, "subtitle", "footnote", etc. creates a predictable data model, where an author can start to do things like write: `about.heading`, trusting that `mh.heading` works.